source/refunding_payments/index.html.md.erb

```diff
@@ -239,70 +114,82 @@ If the payment has reached your bank account, your PSP will take the refund amou
 - the next payment or payments you receive from any of your users, if your PSP is Stripe or ePDQ
 - your bank account, if your PSP is SmartPay or Worldpay
 
-Your PSP will send refunds back to the source account of the original payment. You cannot refund a payment to another card or bank account.
-
 ### Send email notifications about refunds
 
 You can use the GOV.UK Pay admin tool to turn on [sending refund email notifications to your users](https://selfservice.payments.service.gov.uk/email-notifications).
 
-## Search refunds with the API
-
-You can use the GOV.UK Pay API to:
-
-* get information about a single refund
-
-* generate a list of refunds matching search criteria
+## Getting a list of refunds
 
-### Get information about a single refund
+You can use the GOV.UK API to:
 
-`GET /v1/refunds/paymentId`
+- get all refunds for a single payment
+- search refunds
 
-The JSON response body to this API call contains a `"refunds"` object. If the
-refund exists, the `"status"` field will contain one of:
+## Get all refunds for a single payment
 
-* `"submitted"`
+`GET /v1/payments/PAYMENT-ID/refunds`
 
-* `"success"`
+Example response:
 
-* `"error"`
+```javascript
+{
+  "payment_id": "hu20sqlact5260q2nanm0q8u93",
+  ...
+  "_embedded": {
+    "refunds": [
+      {
+        "refund_id": "act4c33g40j3edfmi8jknab84x",
+        "created_date": "2019-09-19T16:52:07.855Z",
+        "amount": 120,
+        "status": "success"
+      },
+      {
+        "refund_id": "9rwjbdcpjd54ol1btog2a8zowh",
+        "created_date": "2019-09-20T20:46:01.121Z",
+        "amount": 60,
+        "status": "success"
+      }      
+    ]
+  }
+}
+```
 
-### Generate a list of refunds (search refunds)
+## Search refunds
 
-`GET /v1/refunds`
+`GET /v1/refunds?<queryparameters>`
 
 An example search request:
 
 `GET
-/v1/refunds?from_date=2018-10-01T13:30:00Z&to_date=2018-10-14T15:00:00Z`
+/v1/refunds?from_date=2019-09-20T13:30:00Z&to_date=2019-09-22T15:00:00Z`
 
-#### Filter by date
+### Filtering by date
 
 You can use the following query parameters to filter refunds by date:
 
-* `from_date` - the start date for payments to be searched, inclusive
-
-* `to_date` - the end date for payments to be searched, exclusive
+* `from_date`, which is the start date to search, inclusive
+* `to_date`, which is the end date to search, exclusive
 
-These take inputs based on a subset of [ISO
-8601](https://www.gov.uk/government/publications/open-standards-for-government/date-times-and-time-stamps-standard).  For example, a valid input would be
-`2015-08-13T12:35:00Z`.
+Dates must be in [ISO
+8601](https://www.gov.uk/government/publications/open-standards-for-government/date-times-and-time-stamps-standard) format. For example `2019-09-20T13:30:00Z`.
 
-#### Pagination
+### Splitting results into pages
 
-You can use the following query parameters for pagination:
+You can use the following query parameters to split results into pages:
 
```

Comment:
since this is tech docs I would keep `pagination` as all devs will understand that, unless there is some guidance I am not aware of